### ISSUE TITLE
Error when creating physical skeleton in TPS demo

### ISSUE THREAD

**Issue by akien-mga** [godotengine/godot#23920]
**Godot version:**
Current master (8ba0d513)

**OS/device including version:**
Mageia 7 x86_64

**Issue description:**
When selecting "Create Physical Skeleton" in the `player.tscn`'s Skeleton of the [TPS demo](https://github.com/godotengine/tps-demo), the following error is printed:
```
ERROR: get_bone_parent: Index p_bone=-1 out of size (bones.size()=112)
   At: scene/3d/skeleton.cpp:481.
ERROR: erase: Condition ' !p_I ' is true. returned: false
   At: ./core/list.h:157.
```

Might be an issue with the model itself, but worth investigating nevertheless.

CC @AndreaCatania 

**Steps to reproduce:**
- Clone [TPS demo](https://github.com/godotengine/tps-demo) and open it (import takes a while, maybe a simpler reproducer can be found) or use MRP below (only `player.tscn` with dae and `.import` folder, 100 MB)
- Open `player.tscn`, select Skeleton, select "Create Physical Skeleton"
- See console

**Minimal reproduction project:**
[`player.tscn` scene only](https://framadrop.org/r/n_rHDOHp5i#YHPrq2ugsblQumAQl9X8E9AT8/UPsxtFmPKomy6YTjg=)

**Comment by akien-mga** [comment]
I added a minimal reproduction project with only the `player.tscn` scene and its dae (and assets pre-imported), takes 100 MB.

**Comment by TheMikirog** [comment]
The issue is still there and it appeared once I ported my project from 3.0 to 3.1
Again, no errors like it in 3.0. Doesn't seem to matter that much, since everything is working, but that error spam is really annoying.
`Index p_bone=-1 out of size (bones.size()=16)`

**Comment by AndreaCatania** [comment]
I didn't checked the code yet, but seems a model problem. Who is managing this project?

**Comment by pouleyKetchoupp** [comment]
The second error is fixed with #34028.

There was a bone attachment on one of the bones that was causing an error when trying to erase bound nodes on bones. For most of them it doesn't trigger the error because the list is empty, and an error was triggered for the only one with a bone attachment.
